src/main/java/nl/tudelft/contextproject/Main.java

```diff
@@ -133,6 +139,20 @@ public void simpleInitApp() {
 	 */
 	protected void setupControlMappings() {
 		inputManager.addMapping("pause", new KeyTrigger(KeyInput.KEY_P));
+		inputManager.addMapping("Left", new KeyTrigger(KeyInput.KEY_A));
+		inputManager.addMapping("Right", new KeyTrigger(KeyInput.KEY_D));
+		inputManager.addMapping("Up", new KeyTrigger(KeyInput.KEY_W));
+		inputManager.addMapping("Down", new KeyTrigger(KeyInput.KEY_S));
+		inputManager.addMapping("Jump", new KeyTrigger(KeyInput.KEY_SPACE));
+	}

```

Comment:
Should use getInputManager() for mocking and testing reasons. (For example, TestUtil properly instantiates getInputManager(), so this method could be tested by it).